torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -2580,7 +2603,59 @@ def test_callback_none(self):
             TypeError,
             "incompatible function arguments."
         ):
-            rpc.rpc_async(dst, raise_func)._then(None)
+            rpc.rpc_async(dst, raise_func).then(None)
+
+    @dist_init
+    @_skip_if_tensorpipe_agent
```

Comment:
Do we know why this doesn't work for tensorpipe? It seems like this high-level futures API should be agent agnostic. Although, we can file an issue for this and not block this PR